testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/management/ServerStartFailureTestCase.java

```diff
@@ -136,10 +136,16 @@ public void testDomainLifecycleMethods() throws Throwable {
         waitUntilState(client, "slave", "failure-three", "DISABLED");
 
         validateResponse(startServer(client, "master", "main-one"));
-        ModelNode result = validateResponse(startServer(client, "slave", "failure-two"));
-        Assert.assertThat(result.asString(), is("FAILED"));
-        result = validateResponse(startServer(client, "slave", "failure-three"));
-        Assert.assertThat(result.asString(), is("FAILED"));
+        ModelNode result = validateFailedResponse(startServer(client, "slave", "failure-two"));
+        String failureDescription = result.asString();
```

Comment:
result.asString() value changed here too, similar as above - comments by James and Brian?

Reply:
@ropalka Any update here?

Reply:
@rsvoboda I agree; this work should not change the semantics of this operation.  Like the other test change I commented on above, it's arguable that the change is valid, but this PR isn't the place to change it. IIRC the op returning the status of the server goes way, way back.